app/views/issues/_table.html.erb

```diff
@@ -32,7 +32,7 @@
                ]
              when 'Affected'
                # this count is coming from the SQL finder in the controller
-               [issue.affected_count, issue.affected_count]
+               [issue.affected_count, issue.affected.pluck(:label).join("\n")]
```

Comment:
This looks like an innocent change, but it's adding an N+1 SQL query. Probably we should load this list in a smarter way in the controller.